pkgs/development/python-modules/gradio/default.nix

```
@@ -140,6 +141,9 @@ buildPythonPackage rec {
   preCheck = ''
     export HOME=$TMPDIR
     cat ${./conftest-skip-network-errors.py} >> test/conftest.py
+  '' + lib.optionalString stdenv.isDarwin ''
+    # OSError: [Errno 24] Too many open files
+    ulimit -n 4096
```

```suggestion
    ulimit -n 1024
```
nit: use same number as linux default.